hardware/ariadne/bootloaders/ariadne/tftp.c

```diff
@@ -37,7 +37,13 @@ const unsigned char tftp_unknown_error_packet[] PROGMEM = "\0\5" "\0\0" "Error";
 #define TFTP_INVALID_IMAGE_LEN 23
 const unsigned char tftp_invalid_image_packet[] PROGMEM = "\0\5" "\0\0" "Invalid image file";
 
-uint16_t lastPacket = 0, highPacket = 0;
+uint8_t tftpFlashing;
+
+#ifndef TFTP_RANDOM_PORT
+static uint16_t tftpTransferPort;
+#endif
+
+static uint16_t lastPacket = 0, highPacket = 0;
```

Comment:
variable in file (.c) scope with static keyword == internal linkage (vs external linkage).
this is just to declare the variable local to the translation unit. 
it is common practice to declare locals as static in the c source, and globals in the header file (with extern keyword)